### PR TITLE
VACMS-2332

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
See #2332 

## Testing done
Visual

## QA steps
As an admin, go to `admin/config/system/taxonomy-entity-index/reindex` and rebuild content index.
Go to `/section/veterans-health-administration/visn-4` and visually verify that content associated with visn-4 (and children) appears

### CODE REVIEW

File: config/sync/views.view.taxonomy_term.yml [hunk 1]
```diff
@@ -1360,249 +1360,6 @@ display:
         title: ''
         description: ''
         weight: 0
-      filters:
```
Comment (swirtSJW) [inline config/sync/views.view.taxonomy_term.yml:1363]: Question:  This change surprises me.   Do you know why all this is being removed and not being replaced with some other config on this view?

Comment (ethanteague) [reply config/sync/views.view.taxonomy_term.yml:1363]: Yep, @swirtSJW - it’s a piece of config that I added in a previous task to winnow content by owner (we had a strange bug where content not associated with owner was showing in certain section views). Upgrading cures this issue, and made this extra filter unnecessary.

Comment (kevwalsh) [reply config/sync/views.view.taxonomy_term.yml:1363]: nice!!!

Comment (swirtSJW) [reply config/sync/views.view.taxonomy_term.yml:1363]: Awesome.   Thanks for clarifying!

**Pull request merged**